can_tools/scrapers/official/SD/sd_vaccines.py

```diff
@@ -99,6 +99,34 @@ def construct_body(self, resource_key, ds_id, model_id, report_id, counties):
                     }
                 }
             },
+            {
+                # TODO(sean): This remove booster shots from request. Currently treating boosters as a subset of
+                # "vaccinations completed" so they are not tracked.
```

Comment:
Wow.  Out of curiosity is there a trick to "writing" these queries or do you do it by hand?

Reply:
For the most part, I just copy the structure of the network calls made by the dashboard and tweak/add any columns or parameters that are needed. 

for example, for this where clause I looked for a query that included a "where not" clause, copied the syntax, and just changed the column and values to what we needed to exclude